napari/_qt/menus/file_menu.py

```diff
@@ -133,46 +30,6 @@ def __init__(self, window: 'Window'):
         self._rebuild_samples_menu()
         self.update()
 
-    def _close_app(self):
-        self._win._qt_window.close(quit_app=True, confirm_need=True)
-
-    def _close_window(self):
-        self._win._qt_window.close(quit_app=False, confirm_need=True)
-
-    def _layer_count(self, event=None):
-        return len(self._win._qt_viewer.viewer.layers)
-
-    def _screenshot_dialog(self):
-        """Save screenshot of current display with viewer, default .png"""
-        hist = get_save_history()
-        dial = ScreenshotDialog(
-            self._win.screenshot, self._win._qt_viewer, hist[0], hist
-        )
-        if dial.exec_():
-            update_save_history(dial.selectedFiles()[0])
-
-    def _open_preferences(self):
-        """Edit preferences from the menubar."""
-        if self._pref_dialog is None:
-            win = PreferencesDialog(parent=self._win._qt_window)
-            self._pref_dialog = win
-
-            app_pref = get_settings().application
-            if app_pref.preferences_size:
-                win.resize(*app_pref.preferences_size)
-
-            @win.resized.connect
-            def _save_size(sz: QSize):
-                app_pref.preferences_size = (sz.width(), sz.height())
-
-            win.finished.connect(self._clean_pref_dialog)
-            win.show()
-        else:
-            self._pref_dialog.raise_()
-
-    def _clean_pref_dialog(self):
-        self._pref_dialog = None
-
     def _rebuild_samples_menu(self):
```

Comment:
Question, I note that this remains here but with regard to this comment:

https://github.com/napari/napari/blob/beaa98efe5cf04ba659086e7a514b2ade05277af/napari/plugins/_npe2.py#L311-L317

how should `_rebuild_samples_menu` change after app-model?
 
cc @DragaDoncila @tlambert03

Reply:
Related the mypy error is
```
mypy napari/settings napari/types.py napari/plugins
napari/plugins/_npe2.py:287: error: "QModelMenu" has no attribute "_rebuild_samples_menu"  [attr-defined]
```

https://github.com/napari/napari/actions/runs/2789075475/jobs/4391972185

Reply:
I don't think we need to deal with this file at all, Isn't the entire file_menu.py to be dropped at this point?